checker/bin-devel/javac

```diff
@@ -21,6 +21,12 @@ esac
 
 binaryDir="${mydir}"/../dist
 
+if [ -z "$LT_BIN" ] || [ "$LT_BIN" = "" ]; then
+   ltBinDir="${mydir}"/../../../jsr308-langtools/cfmake/javac.jar:${binaryDir}/javac.jar
```

Comment:
TODO: this shouldn't refer to jsr308-langtools. Should it refer to the javac from AFU?

Reply:
It's not needed.  (The CI tests pass, but they no longer build jsr308-langtools.)  If we remove it, the following assertion fails though:
https://github.com/typetools/checker-framework/blob/0c7cf8b2febb9692d0337dfadff4392ea3c405b3/framework/src/main/java/org/checkerframework/framework/util/CheckerDevelMain.java#L56-L58

Reply:
How does removing it completely work? Previously there was `tools.jar` on the runtime CP, so I would expect something similar again.
Is the issue maybe that AFU contains a copy of javac and therefore things work, even though we don't have a javac here?
Variable `buildDirs` below uses `annotation-file-utilities-all.jar`. Maybe that should be the slim artifact and here we add javac explicitly?

Reply:
Put differently, what was the point of splitting up `cp` and `runtime.cp`?

Reply:
Thanks!  I'd missed the `${binaryDir}/javac.jar` part.  I've fixed this in 8bfe29c.

Reply:
Also the `runtime.cp` is what ends up on the `bootclasspath`, so we just put `javac.jar` there.